teps/README.md

```diff
@@ -1,185 +1,8 @@
-# Tekton Enhancement Proposals (TEPs)
+# TEPs
```

Comment:
NIT: I think the title here should stay as it was

Reply:
fixed, thanks!